python/graphstorm/model/node_gnn.py

```diff
@@ -110,7 +116,26 @@ def __init__(self, alpha_l2norm):
     def forward(self, blocks, node_feats, _, labels, input_nodes=None):
         """ The forward function for node prediction.
 
-        This GNN model doesn't support edge features for now.
+        This method is used for training. It takes blocks (containing the graph structure),
+        node features, and node labels of a mini-batch as input, and
+        computes the loss of the model in the mini-batch as the return value.
+
+        Parameters
+        ----------
+        blocks : list of DGLBlock
+            The message passing graph for computing GNN embeddings.
+        node_feats : dict of Tensors
+            The input node features of the message passing graphs.
+        _ : This GNN node model doesn't support edge features for now.
+        labels: dict of Tensor
+            The labels of the predicted nodes.
```

Comment:
in format of xxx ?

Reply:
Updated to ` in the format of {target_ntype: labels}.`